SU2_CFD/src/integration_structure.cpp

```diff
@@ -703,27 +695,23 @@ void CIntegration::SetFEM_StructuralSolver(CGeometry *geometry, CSolver **solver
       solver_container[FEA_SOL]->GeneralizedAlpha_UpdateLoads(geometry, solver_container, config);
       break;
   }
-  
+
   /*--- Store the solution at t+1 as solution at t, both for the local points and for the halo points ---*/
-  for (iPoint = 0; iPoint < geometry->GetnPoint(); iPoint++) {
-    
-    solver_container[FEA_SOL]->node[iPoint]->Set_Solution_time_n();
-    solver_container[FEA_SOL]->node[iPoint]->SetSolution_Vel_time_n();
-    solver_container[FEA_SOL]->node[iPoint]->SetSolution_Accel_time_n();
-    
-  }
-  
+
+  solver_container[FEA_SOL]->node->Set_Solution_time_n();
```

Comment:
@rsanfer I saw the name of this method `Set_Solution_time_n` changed slightly can you check I didn't break anything.

Reply:
So remove the underscore from the time_n and time_n1 methods?